src/Symfony/Component/Console/Application.php

```diff
@@ -1169,4 +1169,28 @@ private function splitStringByWidth($string, $width)
 
         return $lines;
     }
+
+    /**
+     * Returns all namespaces of the command name.
+     *
+     * @param string $name The full name of the command
+     *
+     * @return array The namespaces of the command
+     */
+    private function extractAllNamespaces($name)
+    {
+        // -1 as third argument is needed to skip the command short name when exploding
+        $parts = explode(':', $name, -1);
+        $namespaces = array();
+
+        foreach ($parts as $part) {
+            if (count($namespaces)) {
+                $namespaces[] = end($namespaces).":$part";
```

Comment:
we generally don't use interpolation in strings in Symfony (we may have a few places using it though). And we also use single quotes everywhere except when we explicitly need double quotes (for ``\n`` chars for instance)